scrnaseq.Rmd

```diff
@@ -1514,7 +1535,9 @@ All files generated with this report are written into the provided output folder
   + Seurat2Loupe_genesets.csv: Seurat differentially expressed genes   
     Import to Loupe through "Import Lists"   
 * Functional enrichment of differentially expressed genes per cell cluster  
-  + Functions_DEG_down_cluster_1.xlsx, Functions_DEG_up_cluster_1.xlsx, ...: Excel files with one tab per database  
+  + Functions_DEG_down_cluster_1.xlsx, Functions_DEG_up_cluster_1.xlsx, ...: Excel files with one tab per database
+* Cerebro files:
+  + cerebro.crb: Can be loaded into the cerebro interface.
```

Comment:
Remove the dot to make it consistent with the rest of the text. And can you change cerebro -> Cerebro everywhere?

Reply:
Done